src/edu/stanford/nlp/sempre/DateValue.java

```diff
@@ -2,33 +2,40 @@
 
 import fig.basic.LispTree;
 import java.util.Calendar;
+import java.util.Date;
 
 public class DateValue extends Value {
   public final int year;
   public final int month;
   public final int day;
+  public final int hour;
+  public final int minute;
+  public final double second;
 
   // Format: YYYY-MM-DD (from Freebase).
   // Return null if it's not a valid date string.
   public static DateValue parseDateValue(String dateStr) {
-    if (dateStr.equals("PRESENT_REF")) return null;
+    if (dateStr.equals("PRESENT_REF")) return now();
     if (dateStr.startsWith("OFFSET")) return null;
 
     // We don't handle the following things:
     //   - "30 A.D" since its value is "+0030"
     //   - "Dec 20, 2009 10:04am" since its value is "2009-12-20T10:04"
     int year = -1, month = -1, day = -1;
+    int hour = 0, minute = 0;
```

Comment:
For consistency, could we make the defaults -1 to signify "value doesn't exist"?  00:00:00 is actually a valid time.